apps/src/code-studio/components/progress/UnitCalendarDialog.story.jsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import UnitCalendarDialog from './UnitCalendarDialog';
+
+const sampleLessonList = [
```

Comment:
nit: Is this sampleLessonChunkList ?

Reply:
Actually this is a list of lessons, I just have too many fields

Reply:
Ah no worries :)